### PR TITLE
Wrong times showing on the programme builder #4919

### PR DESCRIPTION
**Pull request by naazy**
Wrong times showing on the programme builder #4919

### CODE REVIEW

File: src/TableView.elm [hunk 1]
```diff
@@ -324,15 +331,15 @@ appendFirstRowCell dateWithSessions timeDelimiters model numColumns index column
                     noSessionInDateCellView timeDelimiter dateWithSessions rowSpanVal sessionsInColumn
 
 
-noSessionInDateCellView : Float -> DateWithSessions -> Int -> List Session -> Html Msg
+noSessionInDateCellView : { displayTime : TimeOfDay, delimiter : Float } -> DateWithSessions -> Int -> List Session -> Html Msg
```
Comment (roryc89) [inline src/TableView.elm]: We can use the TimeInfo alias here no?

File: src/TableView.elm [hunk 2]
```diff
@@ -115,17 +120,19 @@ viewDate model numColumns dateWithSessions =
...
-                |> Utils.dropDuplicates
-                |> List.sort
+                |> List.Extra.uniqueBy (\t -> t.delimiter)
+                |> List.sortBy .delimiter
 
         firstTime =
             timeDelimiters
                 |> List.head
-                |> Maybe.withDefault 8
+                |> Maybe.withDefault { delimiter = 8, displayTime = { hour = 0, minute = 0 } }
```
Comment (roryc89) [inline src/TableView.elm]: Does this still display 8am as a default timetable start time when there are no sessions?

**Review by roryc89 — COMMENTED** [review]
Really nice solution. Just a couple of questions

File: src/TableView.elm (hunk 2 above)
Comment (naazy) [reply src/TableView.elm]: Ah is that the expected behaviour? I was a bit confused by the 8. Will change

Comment (naazy) [reply src/TableView.elm]: screen shot 2018-01-11 at 15 52 19

I don't think this is the current behaviour? Or have I misunderstood? Please see above

Comment (naazy) [reply src/TableView.elm]: Have changed the hour to 8 though anyway

**Review by roryc89 — APPROVED** [review]

**Pull request merged**